src/vs/editor/contrib/codeAction/browser/codeActionContributions.ts

```diff
@@ -37,10 +37,10 @@ Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfigurat
 Registry.as<IConfigurationRegistry>(Extensions.Configuration).registerConfiguration({
 	...editorConfigurationBaseNode,
 	properties: {
-		'editor.codeActionWidget.includeNearbyQuickfixes': {
+		'editor.codeActionWidget.includeNearbyQuickFixes': {
```

Comment:
When a setting"s name changes, even only a case-change, shouldn't I also see changes in code that references the setting value, plus migration code to update a user's settings?

Reply:
Completely went over my head - great catch! The fixes/feature changes worked as intended so I didn't think to update that.